Modules/Libs/QuestieLib.lua

```diff
@@ -338,14 +338,16 @@ end
 local cachedTitle = nil
 -- Move to Questie.lua after QuestieOptions move.
 function QuestieLib:GetAddonVersionInfo()
+    local name = "Questie"
+    local version = GetAddOnMetadata(name, "Version")
```

Comment:
Also this should be something like

```suggestion
    local version = GetAddOnMetadata("Questie", "Version") or GetAddOnMetadata("QuestieDev-master", "Version")
```

to also cover the dev version.

Reply:
Fixed. While at it, I added `cachedVersion`.